distributed/scheduler.py

```diff
@@ -657,134 +778,277 @@ class TaskState:
     :      The group of tasks to which this one belongs.
     """
 
+    _key: str
+    _hash: Py_hash_t
+    _prefix: object
```

Comment:
TaskPrefix maybe

Reply:
Since we haven't Cythonized that, it won't have any effect. Also we run into issues as it hasn't been defined yet at this point in the file.

Reply:
I went ahead and put this in quotes, which avoids definition ordering issue. It's still treated as `object` for now though.